azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/function/core/AzureFunctionPackager.java

```diff
@@ -240,7 +240,7 @@ private void copyJarsToStageDirectory(FunctionProject project) throws IOExceptio
     }
 
     private void trackFunctionProperties(Map<String, FunctionConfiguration> configMap) {
-        AzureTelemetry.getActionContext().setProperty(TRIGGER_TYPE, StringUtils.join(getFunctionBindingList(configMap), ","));
+        AzureTelemetry.getContext().getActionParent().setProperty(TRIGGER_TYPE, StringUtils.join(getFunctionBindingList(configMap), ","));
```

Comment:
`getActionParent` or `getParentAction` or `getCurrentAction`?

Reply:
get the parent(actually ancestor) context whose type is `ACTION`